src/sass/style1.scss

```diff
@@ -0,0 +1,291 @@
+  overflow-x: hidden;
+  position: relative;
+}
+html {
+  overflow-x: hidden;
+}
+header {
+  background-color: #f0fafb;
+  .navbar {
+    padding: 1.4rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    .first-nav {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      width: 55%;
+      img {
+        width: 15%;
+        cursor: pointer;
+      }
+      .home,
+      .home:link,
+      .home:visited {
+        cursor: pointer;
+        background-color: white;
+        border-radius: 50%;
+        box-shadow: 0 0 0 black;
+        box-shadow: $base-shadow;
+        img {
+          height: 3rem;
+          width: 3rem;
+          padding: 0.6rem;
+        }
+      }
+      .search-bar {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        width: 75%;
+        background-color: #ffffff;
+        border: 1px solid #e8f2ff;
+        border-radius: 1rem;
+        box-shadow: $base-shadow;
+        padding: 0 0.2rem;
+        .power-off-icon {
+          width: 2.2rem;
+        }
+        img {
+          width: 3rem;
+        }
+        .search {
+          display: flex;
+          gap: 0.2rem;
+          width: 30%;
+          .ij-text {
+            p {
+              font-weight: 500;
+              font-size: 1.4rem;
+              line-height: 2rem;
+              color: #101828;
+            }
+            .subtext {
+              font-weight: 400;
+              font-size: 1.2rem;
+              line-height: 1.8rem;
+              color: #667085;
+              img {
+                width: 1rem;
+                margin-bottom: -0.15rem;
+              }
+            }
+          }
+        }
+      }
+    }
+    .second-nav,
+    .second-nav:hover,
+    .second-nav:link {
+      display: flex;
+      align-items: center;
+      gap: 10%;
+      margin-right: 1rem;
+      cursor: pointer;
+      .arrows {
+        display: flex;
+      }
+      .second-nav-icon {
+        background-color: #d1ebee;
+        padding: 0.5rem;
+        border-radius: 50%;
+        box-shadow: $base-shadow;
+      }
+    }
+    .profile {
+      display: flex;
+      justify-content: space-between;
+    }
+    .second-nav {
+      .one {
+        background-color: #d1ebee;
+        color: #488b92;
+      }
+      .two {
+        background-color: white;
+        color: #01d5eb;
+        margin-left: 0.4rem;
+      }
+    }
+  }
+}
+.configuration {
+  background-color: white;
+  width: 50%;
+  position: relative;
+  margin-inline: auto;
+  border-radius: 1rem;
+  padding-top: 3rem;
+  flex-direction: column;
+  display: flex;
+  justify-content: center;
+  margin-top: 8rem;
+  margin-bottom: 5rem;
+  hr {
+    border-bottom: 1px solid #e4e7ec;
+    margin: 0px 10rem;
+    margin-top: 1rem;
+  }
+  .conf-header {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
```

Comment:
@Zarrouk-Meriem I can't see the child that is absolutely positioned so that this is relatively positioned.